portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String name = request.getParameter("name-input");
+        String email = request.getParameter("email-input");
+        String message = request.getParameter("message-input");
+
+
+
+
+        // Print the value so you can see it in the server logs.
+        System.out.println("Contact: " + name + ", " + email + ", " + message);
+
+        // Write the value to the response so the user can see it.
+        response.getWriter().println("You submitted: " + name + ", " + email + ", " + message);
+
+        response.sendRedirect("http://tpena-sps-spring21.appspot.com/mainPage.html");
```

Comment:
I think you can use a relative url for this one. Something like `response.sendRedirect("mainPage.html");` although I can't remember if this works.